src/internal/components/Toast.test.tsx

```diff
@@ -0,0 +1,239 @@
+import { fireEvent, render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { describe, expect, it, vi } from 'vitest';
+import { Toast } from './Toast';
+
+describe('Toast component', () => {
+  it('bottom-right renders correctly', () => {
```

Comment:
nit: it('should render bottom-right correctly')

Reply:
ack, will fix these up